server/internal/usecase/interactor/item.go

```diff
@@ -160,6 +160,31 @@ func (i Item) Search(ctx context.Context, q *item.Query, sort *usecasex.Sort, p
 	return i.repos.Item.Search(ctx, q, sort, p)
 }
 
+func (i Item) CheckIfItemIsReferenced(ctx context.Context, itemId id.ItemID, correspondingFieldId id.FieldID, _ *usecase.Operator) (*bool, error) {
```

Comment:
```suggestion
func (i Item) CheckIfItemIsReferenced(ctx context.Context, itemId id.ItemID, correspondingFieldID id.FieldID, _ *usecase.Operator) (bool, error) {
```

There is reason to use a pointer bool

Reply:
I wanted to return nil in case of an error, I will return false.
Done!